### PR TITLE
initial package content

### PR DESCRIPTION
**Pull request by pinin4fjords**
This PR establishes the basic infrastructure for CSV file validation. We can establish the specifics of our schema in a future PR.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,91 @@
...
+```bash
+pip install csv_validator
+```
+
+## Usage
+
+To validate a CSV file against the schema stored in the package, use the `csv-validate` command followed by the `--csvfile` option with the path to your CSV file.
+
+```bash
+csv-validate --csvfile path/to/your/csv/file.csv
```
[the remark this answers is not in the document]
Comment (pinin4fjords) [reply README.md:19]: We actually need to do a `pip install -e` to have the entry point work straight off, in some circumstances (including the CI actually). I've added that to the docs

File: README.md [hunk 2]
```diff
@@ -1,2 +1,92 @@
...
+```
+column1,column2,column3
+value1,50,true
+value2,75,false
+value3,25,true
+```
+
+Running the validation command:
+```bash
+csv-validate --csvfile data.csv
```
Comment (luxeredias) [inline README.md:38]: Maybe change here to: 
`csv-validate --csvfile test/test_data_pass.csv`

File: README.md [hunk 3]
```diff
@@ -1,2 +1,92 @@
...
+```
+column1,column2,column3
+value1,150,true
+value2,-10,false
+value3,abc,true
+```
+
+Running the validation command:
+```bash
+csv-validate --csvfile data.csv
```
Comment (luxeredias) [inline README.md:60]: Maybe change here to: 
`csv-validate --csvfile test/test_data_fail.csv`

File: README.md [hunk 4]
```diff
@@ -1,2 +1,92 @@
...
+git clone git@github.com:deeplifeai/csv_validator.git
+pip install -e .
+```
+
+## Usage
+
+To validate a CSV file against the schema stored in the package, use the `csv-validate` command followed by the `--csvfile` option with the path to your CSV file.
+
+```bash
+csv-validate --csvfile path/to/your/csv/file.csv
```
Comment (luxeredias) [inline README.md:19]: the previous alteration to `pip install -e .` worked. I was able to run this command without issues.

**Review by luxeredias — APPROVED** [review]
Everything is working fine on my side.

File: README.md (hunk 3 above)
Comment (pinin4fjords) [inline README.md:60]:
```suggestion
csv-validate --csvfile test/test_data_fail.csv
```

File: README.md [hunk 5]
```diff
@@ -1,2 +1,92 @@
...
+Output:
+```
+Row is valid: {'column1': 'value1', 'column2': '50', 'column3': 'true'}
+Row is valid: {'column1': 'value2', 'column2': '75', 'column3': 'false'}
+Row is valid: {'column1': 'value3', 'column2': '25', 'column3': 'true'}
+```
+
+### Failing Validation
+
+CSV file (`data.csv`):
```
Comment (pinin4fjords) [inline README.md:50]:
```suggestion
CSV file (`test/test_data_fail.csv`):
```

File: README.md (hunk 2 above)
Comment (pinin4fjords) [inline README.md:38]:
```suggestion
csv-validate --csvfile test/test_data_pass.csv
```

File: README.md [hunk 6]
```diff
@@ -1,2 +1,92 @@
...
+csv-validate --csvfile path/to/your/csv/file.csv
+```
+
+## Example
+
+Assuming you have a CSV file named `data.csv` that needs to be validated against the schema stored in the package, here are examples of data that would both pass and fail the validation:
+
+### Passing Validation
+
+CSV file (`data.csv`):
```
Comment (pinin4fjords) [inline README.md:28]:
```suggestion
CSV file (`test/test_data_pass.csv`):
```

File: README.md [hunk 7]
```diff
@@ -1,2 +1,92 @@
...
+
+To validate a CSV file against the schema stored in the package, use the `csv-validate` command followed by the `--csvfile` option with the path to your CSV file.
+
+```bash
+csv-validate --csvfile path/to/your/csv/file.csv
+```
+
+## Example
+
+Assuming you have a CSV file named `data.csv` that needs to be validated against the schema stored in the package, here are examples of data that would both pass and fail the validation:
```
Comment (pinin4fjords) [inline README.md:24]:
```suggestion
Assuming you have passing and failing CSV files named `test_data_pass.csv` and `test_data_fail.csv`, respectively, that need to be validated against the schema stored in the package, here are examples of the pass and failure:
```

**Pull request merged**